omc3/segment_by_segment/propagables/__init__.py

```diff
@@ -0,0 +1,22 @@
+"""
+Propagables
+-----------
+
+In the ``propagables`` module, the parameters that can be propagated
+through the segment, are defined. 
+Each propagable has a corresponding class, which contains the functions that
+describe the forward and backward propagation for the respective parameter.
+
+This module exposes the main classes for propagables, to avoid making the 
+imports so long and complicated.
+"""
+from omc3.segment_by_segment.propagables.abstract import Propagable  # probably only used for type-hinting
+from omc3.segment_by_segment.propagables.utils import PropagableBoundaryConditions   #noqa  -> expose
+from omc3.segment_by_segment.propagables.utils import PropagableColumns   #noqa  -> expose
+from omc3.segment_by_segment.propagables.phase import Phase
+from omc3.segment_by_segment.propagables.beta import BetaPhase
+from omc3.segment_by_segment.propagables.alpha import AlphaPhase
+
+def get_all_propagables() -> tuple[Propagable, ...]:
+    """ Return all defined Propagables. """
+    return Phase, BetaPhase, AlphaPhase
```

Comment:
Feels a bit weird to have a function for three hardcoded things, might as well be a tuple directly.

Reply:
And rather than updating a `return` statement we would just update the tuple?

Reply:
Not that it makes much difference, but as I was also thinking the same I will do that ^^